### DISCUSSION TITLE
İbax Madenciliği

### DISCUSSION THREAD

**Discussion by feromen06** [IBAX-io/go-ibax#1242] [Soru Cevap - Türkçe]
2.1 milyar arzın %57 si madencilik ile çıkacak. 4sn blok ile 25ibxc çıkacak (yanlış anlamış olabilirim)

Bu şekilde toplam arza ulasmak çok ama çok yıllar alır.

Whitepaper da söyle bahsediliyor Bitcoin, Ethereum ve Grin coinleri gibi..... Yani bu 3 coin dışındaki coinlere siktir çekilmiş :) Bence haklılar. 

Ayrıca madencilik yönetiminde Grin den etkilenmişler.

Uyarı: Boş beleş airdrop peşinden koşanlar anlamaz.

**Comment by erhanqwerty** [comment]
sistem gereksinimleri nedir

**Comment by sedat2855** [comment]
UP
